eng/Generate.ps1

```diff
@@ -121,6 +121,10 @@ if (!($Exclude -contains "TestProjects"))
             $inputFile = Join-Path $directory "$testName.json"
             $testArguments ="--require=$configurationPath --input-file=$inputFile"
         }
+        if ($directory -Match "-LowLevel")
```

Comment:
Can we put llc:true into the README.md inside the directory?

Reply:
Our TestServer tests today use raw json not readme, so I'm not sure where?

```
> ls .\test\TestProjects\Accessibility\
Accessibility.csproj  Accessibility.json    Generated
```

Reply:
You can add readme.md and it will be pulled in. See `\test\TestProjects\ModelNamespace\readme.md `